src/js/core.js

```diff
@@ -899,7 +899,7 @@
 
             try {
                 this.events.disableCustomEvent('editableInput');
-                if (opts.url && opts.url.trim().length > 0) {
+                if (opts.url && opts.url.length > 0) {
```

Comment:
So `createLink()` is a public method that can (and is) used by other users externally.  Would leaving the 2 calls to `trim()` from within this `createLink()` method still allow for you to fix the issue with the anchor extension?